cleverhans/utils.py

```diff
@@ -113,6 +118,20 @@ def other_classes(nb_classes, class_ind):
     return other_classes_list
 
 
+def conv_wrap(filters, kernel, strides, padding):
```

Comment:
I'd just name it `conv_2d`, the fact that is a wrapper is an implementation detail that could go in the notes of the docstring but doesn't need to be exposed to the user so prominently.